prostyle.css

```diff
@@ -0,0 +1,130 @@
+nav {
+  width: 100%;
+  height: 50px;
+  text-align: center;
+  font-size: 14px;
+  font-weight: bold;
+  font-family: sans-serif;
+  word-spacing: 50px;
+  margin: auto;
+  background-color: aliceblue;
+  text-decoration: none;
+  padding-top: 20px;
+  float: top;
+}
+
+.backDrop {
+  background-image: url('https://image.shutterstock.com/z/stock-photo-abstract-polygonal-space-low-poly-dark-background-with-connecting-dots-and-lines-connection-550182499.jpg');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: center;
+  width: 100%; height: 1100px;
+}
+
+.profilePic {
+  width: 300px; height: 400px;
+  transform: rotate(0deg);
+  margin-left: 570px;
+  margin-top: 200px;
+  border-style: inset;
+  border-width: thick;
+}
+
+.bio {
+  background-color: darkgray;
+  width: 500px; height: auto;
+  margin-left: 480px;
+  border-style: outset;
+  font-family: serif;
+  font-size: 18px;
+}
+
+.intrests {
```

Comment:
Should be spelt "interests" - be careful with spelling, as you may find yourself unable to target things you think you should be able to.